ember-cli-build.js

```diff
@@ -14,25 +10,25 @@ var EmberApp = require('ember-cli/lib/broccoli/ember-app'),
  * If you update ember-cli and something breaks,
  * the first thing you should try is to comment this out
  */
-EmberApp.prototype.addonTreesFor = function (type) {
-	return this.project.addons.map(function (addon) {
-		if (addon.treeFor) {
-			var tree = addon.treeFor(type);
-
-			if (tree) {
-				// uncomment to see the files available to be filtered out
-				// tree = stew.log(tree, {output: 'tree'});
-				tree = stew.rm(tree,
-					'modules/ember-types/asserts/**/*.js',
-					'modules/ember-types/constants/*.js',
-					'modules/ember-types/property/*.js'
-				);
-			}
-
-			return tree;
-		}
-	}).filter(Boolean);
-};
+// EmberApp.prototype.addonTreesFor = function (type) {
+// 	return this.project.addons.map(function (addon) {
+// 		if (addon.treeFor) {
+// 			let tree = addon.treeFor(type);
+//
+// 			if (tree) {
+// 				// uncomment to see the files available to be filtered out
+// 				// tree = stew.log(tree, {output: 'tree'});
+// 				tree = stew.rm(tree,
+// 					'modules/ember-types/asserts/**/*.js',
+// 					'modules/ember-types/constants/*.js',
+// 					'modules/ember-types/property/*.js'
```

Comment:
There is a lot of unnecessary code in these files. If this way doesn't work anymore, is there something else we can do?